src/styles/shared/components/Buttons/_index.scss

```diff
@@ -0,0 +1,168 @@
+@use "sass:map";
+@use "@hypothesis/frontend-shared/styles/mixins/focus";
+@use './_mixins';
```

Comment:
I think we add a space between the third-party imports and local

Reply:
Yes, I think it would make sense to follow similar import ordering/grouping as for our JS imports.